tests/functional/i/invalid/invalid_all_format.py

```diff
@@ -2,6 +2,6 @@
 
 Tuples with one element MUST contain a comma! Otherwise it's a string.
 """
-__all__ = ("CONST")  # [invalid-all-format]
+__all__ = ("CONST")  # [invalid-all-format, superfluous-parens]
```

Comment:
I'm torn on what to do here, should we make sure the `superfluous-parens` check doesn't fire for the special case of __all__ ?

Reply:
Given the conversation below on the slots case, this code stays as is